src/inspector_agent.cc

```diff
@@ -648,6 +639,92 @@ void Agent::AllAsyncTasksCanceled() {
   client_->AllAsyncTasksCanceled();
 }
 
+void Agent::InterceptAsyncStackDepthMessage(const StringView& message) {
+  // This logic would be better implemented in JavaScript, but when using
+  // --inspect-brk, the debugger must necessarily attach before much JavaScript
+  // can execute. The Debugger.setAsyncCallStackDepth message arrives too early
+  // and we must intercept this in C++.
+  //
+  // TODO: for performance reasons, it would be good to pre-scan the string for
+  // 'Debugger.setAsyncCallStackDepth' and return early if not found.
+
+  v8::Isolate* isolate = parent_env_->isolate();
+  HandleScope handle_scope(isolate);
+  Local<Context> context = parent_env_->context();
+
+  v8::TryCatch try_catch(isolate); // catch and ignore exceptions.
```

Comment:
s/catch/Catch/ and two spaces before the `//`.  (Although I don't think the comment adds much.)